code/modules/jobs/job_types/roguetown/adventurer/types/sorceress.dm

```diff
@@ -0,0 +1,45 @@
+/datum/subclass/sorceress
```

Comment:
Like sentinel, rare pick classes are a bit nessicary. It's Remove it, as it's being a migrant wave, not a job.